### PR TITLE
Do not mute when start playing on sharing slide

### PR DESCRIPTION
**Pull request by swemail**

### CODE REVIEW

File: client/src/routes/Temple/hooks/useMuteAudioListener.spec.tsx [hunk 1]
```diff
@@ -37,14 +41,18 @@ describe('useMuteAudioListener', () => {
       </RecoilRoot>
     );
 
+    mockUseTempleExercise.mockReturnValue({
+      slide: {current: {type: 'reflection'}},
```
Comment (Nipher) [inline client/src/routes/Temple/hooks/useMuteAudioListener.spec.tsx:45]: > it('should toggle audio when state is playing and current slide is not reflection',

should  it not play when reflection? the description doesn't seem to match the assertion 🤔

Comment (swemail) [reply client/src/routes/Temple/hooks/useMuteAudioListener.spec.tsx:45]: Right you are, changed it

**Review by Nipher — APPROVED** [review]
Nice 🤘

**Review by gewfy — APPROVED** [review]
The name of this hook is still a bit vague to me. But let's merge for now!

**Pull request merged**